test/members_page_test.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page.rb'
+
+describe 'MembersPage' do
+  around { |test| VCR.use_cassette('Members Page', &test) }
+
+  describe '#to_h' do
+    subject do
+      url = 'http://sitl.diputados.gob.mx/LXIII_leg/listado_diputados_gpnp.php?tipot=TOTAL'
+      MembersPage.new(response: Scraped::Request.new(url: url).response).to_h
+    end
+
+    it 'should have a :party_groupings field' do
+      subject.to_h[:party_groupings].wont_be_nil
+    end
+  end
+
+  describe 'party groupings' do
+    subject do
+      url = 'http://sitl.diputados.gob.mx/LXIII_leg/listado_diputados_gpnp.php?tipot=TOTAL'
+      MembersPage.new(response: Scraped::Request.new(url: url).response)
+                 .to_h[:party_groupings]
+                 .first
+                 .to_h
+    end
+
+    it 'should contain a party grouping with the correct name' do
+      subject[:name].must_equal 'Partido Revolucionario Institucional'
+    end
+
+    it 'should contain a party grouping with the correct id' do
+      subject[:id].must_equal 'PRI'
+    end
+
+    it 'should contain a party grouping with the correct number of members' do
+      subject[:members].count.must_equal 207
+    end
+  end
+
+  describe 'member row' do
+    subject do
+      url = 'http://sitl.diputados.gob.mx/LXIII_leg/listado_diputados_gpnp.php?tipot=TOTAL'
+      MembersPage.new(response: Scraped::Request.new(url: url).response)
+                 .to_h[:party_groupings].first.to_h[:members].first.to_h
```

Comment:
This seems a little convoluted. Is there a reason you're going through the indirection of `.to_h` there each time, rather than just `.party_groupings.first.members.first`?
`